pkg/infra/pgx.go

```diff
@@ -0,0 +1,82 @@
+package infra
+
+import (
+	"context"
+
+	"github.com/avast/retry-go"
+	"github.com/eleanorhealth/go-common/pkg/errs"
+	"github.com/georgysavva/scany/v2/pgxscan"
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+func PgxPool(ctx context.Context, connString string, traceServiceName string) (*pgxpool.Pool, error) {
```

Comment:
Is this in use in `infra` now?

Reply:
No

Reply:
I don't believe so, but having `pgxpool.Pool` available will be useful for certain applications.